t/lib/t/MusicBrainz/Server/Edit/Medium/SetTrackLengths.pm

```diff
@@ -38,7 +38,7 @@ test all => sub {
     is($medium->tracks->[5]->length, 276933);
     is($medium->tracks->[6]->length, 94200);
 
-    isa_ok exception {
+    ok exception {
```

Comment:
Not sure I follow this change.

Reply:
Not sure I follow it either :D Reverted.